staff/lucas-wloch/challenges/calculadora-app/calculate.js

```diff
@@ -0,0 +1,88 @@
+class calculator {
+    constructor(textAnterior,textActual) {
+        this.textAnterior = textAnterior
```

Comment:
you need to set all semicolons after your statements

Reply:
Done